talkitover_app.py

```diff
@@ -476,8 +502,7 @@ def choose_bot_wordy_response(message, clientId):
                         "i just hate how i look", "i just hate my looks", "i just hate my appearance", "i just hate the way i look",\
                         "i really hate how i look", "i really hate my looks", "i really hate my appearance", "i really hate the way i look",\
                         "i really just hate how i look", "i really just hate my looks", "i really just hate my appearance", "i really just hate the way i look",
-                        "im ugly", "i am ugly"
-                        "i look horrible", "i look disgusting", "i look atrocious"]
+                        "im ugly", "i am ugly", "i look horrible", "i look disgusting", "i look atrocious", "I feel insecure about my body", "I feel insecure about my appearance", "I hate my body", "I despise my body"]
```

Comment:
This as well does not seem to produce the correct response. Again, perhaps because there is a capital letter in the string.